src/VueSimpleIcon.ts

```diff
@@ -78,18 +67,11 @@ export default class VueSimpleIcon extends Vue {
     if (this.size) return this.size;
     return 24;
   }
-
-  parser: DOMParser;
-
-  constructor() {
-    super();
-    this.parser = new DOMParser();
-  }
 }
 
-const renderError = (h: Function): VNode =>
+const renderError = (title: String, h: Function): VNode =>
```

Comment:
@ericcornelissen maybe add size to the error icon?

Reply:
Good idea! Because this required quite a bit of change to the `renderError` function I decided to implement it separately in #21 (which also includes the `title` argument for the function).

Reply:
Thanks for #21